EventsExpress.Db/Entities/ContactAdmin.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using EventsExpress.Db.Enums;
+
+namespace EventsExpress.Db.Entities
+{
+    public class ContactAdmin : BaseEntity
+    {
+        public Guid? SenderId { get; set; }
+
+        public User Sender { get; set; }
+
+        public Guid? AssigneeId { get; set; }
+
+        public User Assignee { get; set; }
+
+        public string Email { get; set; }
+
+        public ContactAdminReason Subject { get; set; }
+
+        public string Title { get; set; }
+
+        public string EmailBody { get; set; }
+
+        public string ResolutionDetails { get; set; }
```

Comment:
I did not see how ResolutionDetails are updated. Are we supposed to update those when we change the status?